dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/util/SystemConfig.java

```diff
@@ -278,6 +278,18 @@ public String getSelectDataverseInfo(Locale locale) {
         return getLocalizedProperty(SettingsServiceBean.Key.SelectDataverseInfo, locale);
     }
 
+    public String getCookieName() {
+        return getLocalizedProperty(Key.CookieName, Locale.ENGLISH);
```

Comment:
There is no point to use `getLocalizedProperty()`. It will cause that both properties could be used: `CookieName` and `CookieName.en`. We can just use `settingsService.getValueForKey()`.

Reply:
Done